### ISSUE TITLE
Sync users from Jumpcloud

### ISSUE THREAD

**Issue by bithost** [DefGuard/defguard#373]
Hello, 
  Is it possible to sync users from Jumpcloud to Defguard using OIDC/SAML/LDAP?  And if yes, please point me to the documentation.
  
  Thanks

**Comment by teon** [comment]
@bithost by "sync" do you mean export your data from Jumpcloud and import them to defguard?
If yes - then I see that JumpCloud has the possibility in the console to export JSON or Export CSV. That would be a great idea to have an import functionality from SaaS solutions - I will look into that and plan this on our release backlog/roadmap - thanks for the idea!

**Comment by bithost** [comment]
@teon , yes. Jumpcloud is my primary source from where i sync users to other apps. So having option to sync users from JC to Defguard would be a great feature.
